### PR TITLE
Fix jabba.rb formula

### PR DESCRIPTION
**Pull request by michaelcusick**
- [ X ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ X ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [ X ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ X ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ X ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ NO ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

The jabba.rb formula installs without error, but doesn't work. This PR fixes that issue. A similar PR will be opened on homebrew-core.

See: https://github.com/shyiko/jabba/issues/138

### CODE REVIEW

File: Formula/jabba.rb [hunk 1]
```diff
@@ -4,16 +4,16 @@ class Jabba < Formula
...
 
   bottle do
     cellar :any_skip_relocation
     rebuild 2
-    sha256 "a730868e347cb89b50393f96062b2c076f1d501b137ead1a795aa39e54ad4611" => :big_sur
-    sha256 "7eddb409c7bb2784db21756e624a18b19977bb4df53ab547eaedd8abe876651e" => :catalina
-    sha256 "3101ea25ce49c3ed96b3c8595a5441fec3aeb536b56eca21c1dea56f6c1fd86b" => :mojave
-    sha256 "8454f5aa9b8832908b1c889531118ea058b2e675ef7f7f37eeb282f454aeec1e" => :high_sierra
-    sha256 "237bab11bbcc4434a0b48c8d03a6f8634d12ec011f9357f26d8965f523fedad2" => :x86_64_linux
+    sha256 "9a971099c0ab85804166dbf96b99653f37cc8095749df982d346ee397c8f9cd5" => :amd64_linux
```
Comment (iMichka) [inline Formula/jabba.rb:13]: Don't modify the Mac bottles lines.

Just remove the linux bottle she, that's enough.

File: Formula/jabba.rb [hunk 2]
```diff
@@ -4,16 +4,16 @@ class Jabba < Formula
   url "https://github.com/shyiko/jabba/archive/0.11.2.tar.gz"
   sha256 "33874c81387f03fe1a27c64cb6fb585a458c1a2c1548b4b86694da5f81164355"
   license "Apache-2.0"
+  revision 1
```
Comment (iMichka) [inline Formula/jabba.rb:7]: revision 1 unless OS.mac?

File: Formula/jabba.rb [hunk 3]
```diff
@@ -30,13 +30,15 @@ def install
       system "go", "build", "-ldflags", ldflags, "-o", bin/"jabba"
       prefix.install_metafiles
     end
+    mkdir("$HOME/.jabba")
```
Comment (iMichka) [inline Formula/jabba.rb:33]: Do you know why this is not needed on Mac?
Why are we installing something to the users home folder?

**Review by iMichka — CHANGES_REQUESTED** [review]
Hi. Thanks for working on this. I made a few comments

**Comment by michaelcusick** [comment]
Good questions. I opened this PR too hastily. Let me check back with the author of jabba to see what needs to be done to fix the brew packages. Right now jabba doesn't work on Mac or Linux when installed with brew because it is missing some shell scripts. When using the shell script installer, those scripts get installed to `$HOME/.jabba`. I'm probably not doing this correctly. Apologies.

**Pull request closed without merging**